services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/ThrottlingRerouteSpec.groovy

```diff
@@ -294,6 +294,37 @@ class ThrottlingRerouteSpec extends BaseSpecification {
         db.countFlows() == 0
     }
 
+    def "Auto-reroute on the flow which is already deleted should not revive the flow"() {
+        given: "A flow"
```

Comment:
Would be nice to change to smth like "A new flow is created" - Spock reports would look nicer.